examples/time/time.bal

```diff
@@ -34,12 +34,12 @@ public function main() returns error? {
     time:Seconds utcDiff = time:utcDiffSeconds(currentUtc, utc1);
     io:println(`UTC diff: ${utcDiff}`);
 
-    // Check the given `time:Date` value is valid or not.
+    // Checks the given `time:Date` value is valid or not.
```

Comment:
```suggestion
    // Checks if the given `time:Date` value is valid or not.
```

Reply:
Fixed in here https://github.com/ballerina-platform/ballerina-distribution/pull/1674/commits/7474f8ba8d119904de969e213366674d1dec8383